Gemfile

```diff
@@ -7,7 +7,7 @@ gem 'gettext', '>= 3.1.3', '< 4.0.0'
 
 group :test do
   gem 'ci_reporter_minitest', '~> 1.0.0', require: false
-  gem 'minitest', '5.14.1'
+  gem 'minitest', '~> 5.18.0'
```

Comment:
`>= 5.18.0 < 5.19.0` Yeah that was my point, but you are right that it can be safely pinned to `>=5.18`, seeing the change log of minitest they really don't introduce breaking changes.